### ISSUE TITLE
A11y Guide updates

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#403]
The following updates should be added to the A11y Guide:

- Include a statement about current expectations for keyboard interaction
  - also consider if the A11y guide is the right place to include a statement about keyboard interaction documentation (i.e. describing what user actions trigger what changes to the component), or if this belongs in another guidelines document
- Include a note in the Testing section about checking for focus indication.
- Replace the WCAG guideline regarding size with our own standard (i.e. 32px instead of 44px)

**Comment by jgiardino** [comment]
I am not including any statements about keyboard accessibility at this time. I think we should hold off on making any statements until we have implemented and tested at least one react component for one of the widgets listed in the [wai-aria authoring practices](https://www.w3.org/TR/wai-aria-practices-1.1/).

I did make other updates listed in this issue in #452
